plugins/woocommerce-admin/client/homescreen/mobile-app-modal/style.scss

```diff
@@ -154,7 +154,16 @@
 	}
 }
 
-.woocommerce-stepper.is-vertical .woocommerce-stepper__step.is-complete {
+.login-stepper-wrapper {
+	button.install-app-button {
```

Comment:
if you have variant="primary" on the button props you won't have to handle the colouring - the color will automatically match the theme for the WP installation
E.g:
```
<Button variant="primary">text</Button>
```

You can find the @wordpress/components storybook here: https://wordpress.github.io/gutenberg/?path=/story/components-button--primary